exampleCourse/questions/demo/autograder/python/mustacheProcess/tests/ans.mustache.py

```diff
@@ -0,0 +1,8 @@
+{{#params}}
+def {{function_name}}(input):
+    {{#pairs}}
+    if input == "{{input}}":
+        return "{{output}}"
+    {{/pairs}}
+    return {{default_output}}
+{{/params}}
```

Comment:
Seeing GitHub try and fail to deal with the syntax here, I'm wondering if we should mandate that `.mustache` is at the end of a filename, rather than in the middle of it? Is there a specific use case for `*.mustache.*`?

Reply:
I don't know what the right choice is here, but I can tell you why I allowed the `*.mustache.*` version: I don't have any mustache syntax highlighting/checking in my editor. So, it's the language dialect that does a better job for me. If I put `.mustache.` in the middle, that means I get as much Python (or whatever language) highlighting as the editor can give me.

To make this work a little better, I try to avoid mustache tags outside of strings and comments where that's clean to do. I've updated my files to do this (but there are still 'visible' mustache tags because of things like identifier names or unquoted values).

Thoughts?

I guess the perfect world solution would be to get some sort of hybrid of mustache and language-specific formatting, but I don't think I'm going to get there.